src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -373,11 +377,11 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
       status match {
         case Run.Success if backendCall.call.failOnStderr && stderrLength.intValue > 0 =>
-          FailedExecution(new Throwable(s"${makeTag(backendCall)} execution failed: stderr has length $stderrLength"))
+          FailedExecution(new Throwable(s"${log.makeTag} execution failed: stderr has length $stderrLength"))
```

Comment:
In fact, it seems like "makeTag" is a warning sign that the workflow-specific logger will be skipped?